tensorflow/lite/kernels/reverse.cc

```diff
@@ -52,10 +56,21 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
     return kTfLiteError;
   }
 
-  // TODO(renjieliu): support multi-axis case.
-  if (NumElements(axis) > 1) {
-    context->ReportError(context, "Current does not support more than 1 axis.");
-  }
+  // Allocate temporary tensor which is required in kernel operation
+  int temp_tensor_id = 0;
+  context->AddTensors(context, 1, &temp_tensor_id);
+
+  TfLiteIntArrayFree(node->temporaries);
+  node->temporaries = TfLiteIntArrayCreate(1);
+  node->temporaries->data[0] = temp_tensor_id;
+
+  TfLiteTensor* input_copy = GetTemporary(context, node, /*index=*/0);
```

Comment:
I'm thinking we should bump the op version if we know that the axis tensor is multi-dimensional. Other than that I think this looks good.